### PR TITLE
Clarify PII concerns in the explainer.

### PR DESCRIPTION
**Pull request by snianu**
Closes https://github.com/MicrosoftEdge/MSEdgeExplainers/issues/542

### CODE REVIEW

**Comment by snianu** [comment]
@caraitto Addressed your comment. PTAL.

File: docs/clipboard-pickling/explainer.md
Comment (caraitto) [inline docs/clipboard-pickling/explainer.md]: nit: is less visible/obvious -> may be less visible/obvious.

**Review by caraitto — APPROVED** [review]
Thanks for making these changes!

File: docs/clipboard-pickling/explainer.md
Comment (snianu) [reply docs/clipboard-pickling/explainer.md]: Done.

**Comment by snianu** [comment]
@travisleithead Need an LGTM from someone who has write access. Could you please take a look and let me know if you have any concerns?

**Comment by snianu** [comment]
@travisleithead Gentle ping..

**Comment by caraitto** [comment]
@travisleithead Hi, friendly ping?

**Review by BoCupp-Microsoft — APPROVED** [review]

**Pull request merged**
